libs/ui-lib/lib/common/api/assisted-service/UISettingsAPI.ts

```diff
@@ -0,0 +1,24 @@
+import { clientWithoutCaseConverter as client } from '../../api/axiosClient';
+import { Cluster } from '@openshift-assisted/types/./assisted-installer-service';
+
+const config = {
+  headers: {
+    'Content-Type': 'application/json',
+  },
+};
+
+const UISettingsAPI = {
+  makeBaseURI(clusterId?: Cluster['id']) {
+    return `/v2/clusters/${clusterId ? clusterId : ''}/ui-settings`;
```

Comment:
Why is clusterId optional? You cannot call this API if you don't know the cluster ID...